templates/security/notices/lsn.html

```diff
@@ -0,0 +1,139 @@
+
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-12">
+	    <h1>{{ notice.id }}: {{ notice.title }}</h1>
+      <p class="p-muted-heading">{{ notice.published.strftime("%d %B %Y") }}</p>
+      <p>{{ notice.summary | safe }}</p>
+      <h2>Releases</h2>
+      <ul class="p-inline-list u-no-margin--bottom">
+        {% for release in notice.releases %}
+          <li class="p-inline-list__item">
+            <a href="/security/notices?release={{ release.codename }}" class="p-button--tag">Ubuntu {{ release.version }} {{ release.support_tag }}</a>
+          </li>
+        {% endfor %}
+      </ul>
+    </div>
+  </div>
+
+  {% if notice.package_descriptions %}
+  <div class="row">
+    <div class="col-12">
+      <h2>Packages</h2>
+      <ul class="p-list">
+        {% for package_name, package_title in notice.package_descriptions.items() %}
+          <li class="p-list__item"><a href="/security/cve?package={{ package_name }}">{{ package_name }}</a> - {{ package_title }}</li>
+        {% endfor %}
+      </ul>
+    </div>
+  {% endif %}
+
+  <div class="row">
+    <div class="col-8">
+      <h2>Details</h2>
+      <p>{{ notice.details|safe }}</p>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-8">
+      <h2>Checking update status</h2>
+      <p>The problem can be corrected in these Livepatch versions:</p>
+      <table>
+        <thead>
+          <th>Kernel type</th>
+          {% for version in notice.instructions %}
+            <th>{{ version }}</th>
+          {% endfor %}
+        </thead>
+        <tbody>
+          {% for instruction_package in notice.instruction_packages %}
+            <tr>
+              <td><strong>{{ instruction_package }}</strong></td>
+              {% for version in notice.instructions %}
+                {% if notice.instructions[version][instruction_package] %}
+                  <td>{{ notice.instructions[version][instruction_package] }}</td>
+                {% else %}
+                  <td>&mdash;</td>
+                {% endif %}
+              {% endfor %}
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+      <p>To check your kernel type and Livepatch version, enter this command:</p>
+      <div class="p-code-copyable">
+        <input class="p-code-copyable__input" value="canonical-livepatch status" readonly="readonly">
+        <button class="p-code-copyable__action">Copy to clipboard</button>
+      </div>
+    </div>
+  </div>
+
+  {% if notice.release_packages %}
+    <div class="row">
+      <div class="col-8">
+        <h2>Support Information</h2>
+        <p>Kernels older than the levels listed below do not receive livepatch updates. If you are running a kernel version earlier than the one listed below, please upgrade your kernel as soon as possible.</p>
+        {% for version in notice.release_packages.__reversed__() %}
+          <h5>Ubuntu {{ version }}</h5>
```

Comment:
Descending headings should step down sequentially, the previous heading was a `h2`, so this should be a `h3` - but I guess you need the `h5` style, so you could use:
```suggestion
          <h3 class="p-heading--five">Ubuntu {{ version }}</h3>
```

Reply:
Fixed! @sowasred2012